packages/dds/merge-tree/src/test/revertibleFarm.spec.ts

```diff
@@ -0,0 +1,182 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
+import random from "random-js";
+import { SegmentGroup } from "../mergeTreeNodes";
+import {
+    appendToMergeTreeDeltaRevertibles,
+    MergeTreeDeltaRevertible,
+    revertMergeTreeDeltaRevertibles,
+} from "../revertibles";
+import {
+    removeRange,
+    doOverRange,
+    generateOperationMessagesForClients,
+    applyMessages,
+    annotateRange,
+} from "./mergeTreeOperationRunner";
+import { createRevertDriver } from "./testClient";
+import { createClientsAtInitialState, TestClientLogger } from "./testClientLogger";
+
+ const defaultOptions = {
+    initialOps: 5,
+    minLength: { min: 1, max: 256, growthFunc: (i) => i * i },
```

Comment:
i want to grow the min text size faster, so we get deep trees with lots of segments.